content/docs/start/data-pipelines.md

```diff
@@ -1,13 +1,23 @@
+---
+description: 'Learn how DVC data pipelines can help you organize, version, and
+efficiently reproduce your data science or machine learning project.'
+---
+
 # Data Pipelines
 
 Versioning large data files and directories for data science is great, but not
 enough. How is data filtered, transformed, or used to train ML models? DVC
 introduces a mechanism to capture _data pipelines_ — series of data processes
 that produce a final result.
 
-DVC pipelines and their data can also be easily versioned (using Git). This
-allows you to better organize your project, and reproduce your workflow and
-results later exactly as they were built originally!
+A DVC pipeline uses stages to describe how models and other data artifacts are
+built, and provides an efficient way to reproduce them. This allows you to
+capture simple data workflows, organize data science projects, or build detailed
+machine learning pipelines (to name a few use cases).
+
+You can easily version DVC pipelines and their data (using Git). This allows you
+to better organize your project and reproduce your workflow and results later —
+exactly as they were built originally!
```

Comment:
The thing with Get Started pages is that we don't want to get in the way of the hands-on sections so 3 paragraphs for intro is probably too much. In this case I think inserting the target terms (machine learning pipelines, efficient?) in existing text would be better.

Not sure we need to expand on use cases here anyway, we have /doc/use-cases for that. Maybe link to the appropriate use case instead (if not linked yet).

Reply:
As mentioned in the PR description (and discussed offline), "data pipelines" and "machine learning pipelines" are high potential terms which dvc.org does not yet rank well for. 

This doc is a good example of "lowest hanging fruit" (like you mentioned in #1856). It has the most relevant content for the above terms — matching the intent and use case for both. 

The expanded intro is a good way to add the key terms (necessary) and explain the context to people who might be visiting DVC for the first time (warranted, in lieu of other pipeline content). 

> Maybe link to the appropriate use case instead (if not linked yet).

What would that be for pipelines? Aside from this doc, the next best candidates I see are the command references for `dag` and `repro`. Am I missing something?

Reply:
> This doc is a good example of "lowest hanging fruit" (like you mentioned in #1856). It has the most relevant content for "data pipelines" and "machine learning pipelines" (high potential terms which dvc.org does not yet rank well for).

Agree, let's definitely target them here (and then in other related docs as well).

> The expanded intro is a good way to add the key terms (necessary) and explain the context to people who might be visiting DVC for the first time

We want to add "machine learning pipelines", yes. But we don't have to create a whole new paragraph for that. Can we try to find an alternative place to insert it instead?

The part about explaining more context seems out of scope for this PR to me. We could discuss that separately (feel free to open a separate issue) but the new text, esp. "A DVC pipeline uses stages to describe..." is already implicit when you read the doc. (GS pages try to be more demonstrative than descriptive).

> the next best candidates I see are the command references for dag and repro

As for where else to optimize these terms, indeed dat and repo are good, as well as `run` but probably the more impactful one may be in https://dvc.org/doc/user-guide/dvc-files-and-directories#dvcyaml-file

Reply:
Posted a possible resolution in https://github.com/iterative/dvc.org/pull/1857#pullrequestreview-512241072 below.

Reply:
> Posted a possible resolution in #1857 (review) below.

Nice! I like how you were able to condense it and preserve the meaning.

It's a good idea to address this particular term "machine learning pipelines" and add context across `dag` `repro` `run` and [files and directories](https://dvc.org/doc/user-guide/dvc-files-and-directories#dvcyaml-file). I'll take you up on that and create a separate issue for it.

Reply:
Thanks. Left a small follow up about that in https://github.com/iterative/dvc.org/pull/1857#pullrequestreview-516465031, actually.

> I'll take you up on that and create a separate issue for it.

Please do! Resolving here.